dev-server/documents/html/vitalsource.mustache

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>VitalSource book reader</title>
+    <style>
+      body {
+        font-family: sans-serif;
+      }
+    </style>
+    <script type="module">
+      /**
+       * Custom element that holds the book content frame. The class name is not
+       * important but the `<mosaic-book>` tag name triggers the VitalSource integration.
```

Comment:
Note there currently is no VitalSource integration, but it is coming soon.

Reply:
This implementation mimicks the essential elements of the VS UI that affect how the Hypothesis integration works. This is all written from scratch, nothing is taken from elsewhere except the name of the custom element.